nbs/examples/MLFlow_and_neuralforecast.ipynb

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.        forecast = nf.predict(futr_df=Y_test_df)
</pre><p>You can just directly assign Y_hat_df here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Removed redundant step as suggested (via commit https://github.com/Nixtla/neuralforecast/pull/892/commits/33a1de20db576e2aee7d0f83a785568b2f6a3d0f).